state/factory/factory.go

```diff
@@ -471,6 +489,9 @@ func (sf *factory) PutBlock(ctx context.Context, blk *block.Block) error {
 		)
 	}
 
+	if blk.Height() == sf.stopHeight && !sf.commitBlock {
```

Comment:
if `blk.Height() == sf.stopHeight` and `sf.commitBlock == true`, block will be committed, and move to next height.
If we want to stop at height `h`, and then commit block `h`, we can easily update `stopHeight` to `h+1`

Reply:
I see. `commit`  command is useless, and delete it